spec/support/features/document_capture_step_helper.rb

```diff
@@ -34,6 +29,9 @@ def attach_liveness_images(
     )
   )
     attach_images(file)
+    if IdentityConfig.store.doc_auth_separate_pages_enabled
+      click_continue
+    end
```

Comment:
these 3 lines appear to be the only necessary changes in this file 👍🏿

Reply:
@amirbey should be resolved now, I still kept the 'remove continue_doc_auth_form' change